### ISSUE TITLE
bug(?): SC2086 is not available on koalaman/shellcheck/wiki

### ISSUE THREAD

**Issue by himkt** [koalaman/shellcheck#2957]
Hello, I just came shellcheck Wiki from GitHub docs (https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#good-practices-for-mitigating-script-injection-attacks), and found SC2086 is not available on the shellcheck Wiki.

https://github.com/koalaman/shellcheck/wiki/SC2086 navigated me to the `create` page (note that, however, I can see the page on https://www.shellcheck.net [https://www.shellcheck.net/wiki/SC2086] 🤔).

screenshot of https://github.com/koalaman/shellcheck/wiki/SC2086:
<img src="https://github.com/koalaman/shellcheck/assets/5164000/f6b7ba45-e853-4b12-93ad-b15c88558d38" width="70%" />

Moreover, I'm not sure why but SC2086 appeared in the search result.
https://github.com/search?q=repo:koalaman/shellcheck+SC2086&type=wikis
<img src="https://github.com/koalaman/shellcheck/assets/5164000/6529d7f8-1248-4216-9bde-d00ca6b16265" width="70%" />

So, my report is that, I'm not sure why, but SC2086 is unavailable on GitHub Wiki. 

#### For bugs
- Rule Id: SC2086
- My shellcheck version (`shellcheck --version` or "online"): online (Wiki)
- ~[ ] ~The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)~
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/your/interpreter
your script here

```

**Comment by brother** [comment]
Not all error codes have pages (yet). The wiki is open for editing and adding a missing page is very welcome. 

See also #2572 and #2943 - closing this.

**Comment by brother** [comment]
Fat fingered that one indeed.

Someone deleted the page. Let's reinstate it.

Looking at the git log  for that page it has seen some vandalism.

**Comment by ajames-csu** [comment]
This page appears to have been deleted again. The version on shellcheck.net is still available, the "See this page on github" link leads to a 404/create new page form.

**Comment by brother** [comment]
Done.
